doc/Pacemaker_Explained/en-US/Ch-Rules.txt

```diff
@@ -710,6 +825,62 @@ Rules may be used similarly in +instance_attributes+ or +utilization+ blocks.
 Any single block may directly contain only a single rule, but that rule may
 itself contain any number of rules.
 
++rsc_expression+ and +op_expression+ blocks may additionally be used to set defaults
+on either a single resource or across an entire class of resources with a single
+rule.  +rsc_expression+ may be used to select resource agents within both +rsc_defaults+
+and +op_defaults+, while +op_expression+ may only be used within +op_defaults+.  If
+multiple rules succeed for a given resource agent, the last one specified will be
+the one that takes effect.  As with any other rule, boolean operations may be used
+to make more complicated expressions.
+
+.Set all IPaddr2 resources to stopped
+=====
+[source,XML]
+-------
+<rsc_defaults>
+    <meta_attributes id="op-target-role">
+        <rule id="op-target-role-rule" score="INFINITY">
+            <rsc_expression id="op-target-role-expr" class="ocf" provider="heartbeat"
+              type="IPaddr2"/>
+        </rule>
+        <nvpair id="op-target-role-nvpair" name="target-role" value="Stopped"/>
+    </meta_attributes>
+</rsc_defaults>
+-------
+=====
+
+.Set all monitor action timeouts to 7 seconds
+=====
+[source,XML]
+-------
+<rsc_defaults>
+    <meta_attributes id="op-monitor-defaults">
+        <rule id="op-monitor-default-rule" score="INFINITY">
+            <op_expression id="op-monitor-default-expr" name="monitor"/>
+        </rule>
+        <nvpair id="op-monitor-timeout" name="timeout" value="7s"/>
+    </meta_attributes>
+</rsc_defaults>
+-------
+=====
+
+.Set the monitor action timeout on all IPaddr2 resources with a given monitor interval to 7 seconds
+=====
+[source,XML]
+-------
+<op_defaults>
+    <meta_attributes id="op-monitor-and">
+        <rule id="op-monitor-and-rule" score="INFINITY">
+            <rsc_expression id="op-monitor-and-rsc-expr" class="ocf" provider="heartbeat"
+              type="IPaddr2"/>
+            <op_expression id="op-monitor-and-op-expr" name="monitor" interval="10s"/>
+        </rule>
+        <nvpair id="op-monitor-and-timeout" name="timeout" value="8s"/>
+    </meta_attributes>
+</op_defaults>
```

Comment:
8 seconds :)

Reply:
Hah, fixed.  Too much copy and paste going on here.